src/App.tsx

```diff
@@ -1,35 +1,28 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+// import { useState } from "react";
+// import "./App.css";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import { Root } from "./Root";
+import { Homepage } from "./Homepage";
+import { Filming } from "./Filming";
+import { Photography } from "./Photography";
+import { EquipmentRental } from "./EquipmentRental";
 
-function App() {
-  const [count, setCount] = useState(0)
+const router = createBrowserRouter([
+	{
+		path: "/",
+		element: <Root />,
+		children: [
+			{ path: "/", element: <Homepage /> },
+			{ path: "/filming", element: <Filming /> },
+			{ path: "/photography", element: <Photography /> },
+			{ path: "/equipment-rental", element: <EquipmentRental /> },
+			// { path: "/calendar", element: <Calendar /> },
+		],
+	},
+]);
```

Comment:
Jen pro info: Router se většinou dává do separátního `router.tsx` souboru, protože má tendenci se dost rozrůstat a dělá v `App.tsx` bordel, ale tady to takový problém nebude.

Reply:
Zkusila jsem to oddělit. Tak snad dobře.

Reply:
Ok, opravila jsem, co šlo. Fungovalo mi i Object.entries, ale jen do cvhíle, než jsem se skrze něj snažila přidat i componenty. Nepíše mi to chybu, ale stránka nic nezobrazí. A taky nevím, jak se zbavit podtržení u keys. Poradíš?

Reply:
Nechala jsem tam části kódu před těmi posledními změnami (tahání komponentů z globalVariables) v komentářích.

Reply:
Mně to normálně funguje 🤷‍♂️
<img width="328" alt="Screenshot 2024-11-19 at 16 05 30" src="https://github.com/user-attachments/assets/ca6c2adb-9be7-43a1-b9a0-bfcbdcfa2508">

A ten error s key vyřeším buď tím, že pak zavoláš `void key` a nebo tam můžeš dát `_` místo `key`, který se bere, jako že tu hodnotu nechceš použít.